### ISSUE TITLE
Terminal "Can't retrieve terminal details" is not a valid TTY

### ISSUE THREAD

**Issue by l80** [php-school/learn-you-php#53]
``` shell
$ sudo learnyouphp
```

``` shell
PHP Fatal error:  Uncaught exception 'PhpSchool\CliMenu\Exception\InvalidTerminalException' with message 'Terminal "Can't retrieve terminal details" is not a valid TTY' in /Users/l80/.composer/vendor/php-school/cli-menu/src/CliMenu.php:121
Stack trace:
#0 /Users/l80/.composer/vendor/php-school/cli-menu/src/CliMenu.php(98): PhpSchool\CliMenu\CliMenu->assertTerminalIsValidTTY()
#1 /Users/l80/.composer/vendor/php-school/cli-menu/src/CliMenu.php(317): PhpSchool\CliMenu\CliMenu->configureTerminal()
#2 /Users/l80/.composer/vendor/php-school/php-workshop/src/Command/MenuCommand.php(32): PhpSchool\CliMenu\CliMenu->open()
#3 [internal function]: PhpSchool\PhpWorkshop\Command\MenuCommand->__invoke('/usr/local/bin/...')
#4 /Users/l80/.composer/vendor/php-school/php-workshop/src/CommandRouter.php(169): call_user_func_array(Object(PhpSchool\PhpWorkshop\Command\MenuCommand), Array)
#5 /Users/l80/.composer/vendor/php-school/php-workshop/src/CommandRouter.php(153): PhpSchool\PhpWorkshop\CommandRouter->callCommand in /Users/l80/.composer/vendor/php-school/cli-menu/src/CliMenu.php on line 121
```

``` shell
$ sw_vers -productVersion 
10.11.3
```

**Comment by l80** [comment]
Posix was not installed
`sudo port install php56-posix`

**Comment by AydinHassan** [comment]
@l80 we should probably handle this better, @mikeymike is there something we should do here? Can we require posix ext with Composer?

**Comment by mikeymike** [comment]
Yeah we should require it... It needs a general cleanup tbh... removing the Windows terminal class and just roll a single terminal class 

Obviously we hoped to support Windows separately but I don't think we'll be able to.

**Comment by AydinHassan** [comment]
This is fixed in #70
